plugins/ssh/komand_ssh/actions/run/schema.py

```diff
@@ -59,37 +58,41 @@ class RunOutput(komand.Output):
     }
   },
   "required": [
-    "results"
+    "all_output",
+    "results",
+    "stderr",
+    "stdout"
   ],
   "definitions": {
     "results": {
       "type": "object",
       "title": "results",
       "properties": {
-        "all_output": {
+        "stdout": {
           "type": "string",
-          "title": "All Output",
-          "description": "All output",
-          "order": 3
+          "title": "STDOUT",
+          "description": "Stdout",
+          "order": 1
         },
         "stderr": {
           "type": "string",
           "title": "STDERR",
           "description": "Stderr",
           "order": 2
         },
-        "stdout": {
+        "all_output": {
           "type": "string",
-          "title": "STDOUT",
-          "description": "Stdout",
-          "order": 1
-        }
-      },
-      "required": [
-        "all_output",
-        "stderr",
-        "stdout"
-      ]
+          "title": "All Output",
+          "description": "All output",
+          "order": 3
+        },
+        "required": [
+          "all_output",
+          "stderr",
+          "stdout",
+          "results"
+        ]
+      }
```

Comment:
```suggestion
        }
      }
      "required": [
          "all_output",
          "stderr",
          "stdout"
        ]
```
results should be removed and this should be moved up on level.